tests/test_job_router.py

```
@@ -124,9 +117,49 @@ def test_successful_route_returns_worker_id():
     print("PASSED")
 
 
+def test_route_does_not_query_db_for_worker_lookup():
+    print("\n--- TEST 5: route() doesn't hit the DB to look up worker address/port ---")
+    router, ring = _router_with_one_worker()
```

Ruff (0.16.1)

[warning] Unpacked variable `ring` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)